cmd/log.go

```diff
@@ -0,0 +1,24 @@
+package cmd
+
+import (
+	"io"
+	"os"
+
+	"github.com/sirupsen/logrus"
+)
+
+var logger *logrus.Logger
+
+func init() {
+	logger = logrus.New()
+
+	logFile, err := os.OpenFile("preflight.log", os.O_CREATE|os.O_WRONLY|os.O_APPEND, 0700)
```

Comment:
I would probably add a timestamp to make sure subsequent execution will not overwrite the previous logs.

Reply:
Done 👍